### DISCUSSION TITLE
Disable color form rgba to hex

### DISCUSSION THREAD

**Discussion by Alvinnn1** [tailwindlabs/tailwindcss#9011] [Help] [answered]
**What version of Tailwind CSS are you using?**

For example: v3.1.2

**What build tool (or framework if it abstracts the build tool) are you using?**

For example: react@18 vitejs@2.8.6

**What version of Node.js are you using?**

For example: v16.16.0

**What browser are you using?**

For example: Chrome

**What operating system are you using?**

For example: macOS 12.1

**Describe your issue**

Describe the problem you're seeing, any important steps to reproduce and what behavior you expect instead
1. When i define some custom color with rgba in tailwind.config.js
```
module.exports = {
  content: [
    // Example content paths...
    './public/**/*.html',
    './src/**/*.{js,jsx,ts,tsx,vue}',
  ],
  theme: {
    extend: {
      colors: {
        'font-primary': 'rgba(51, 51, 64, 0.88)',
        'font-secondary': 'rgba(71, 71, 101, 0.55)',
        'font-tertiary': 'rgba(58, 58, 89, 0.33)',
        'font-quarternary': 'rgba(61, 61, 89, 0.18)',
      },
    },
  },
}
```
2. Use in jsx
`<p className="text-font-quarternary">User</p>`
3. Work as expect in develop env
![2967AE02-266B-41B5-9B63-DD2A7E8E50BB](https://user-images.githubusercontent.com/13961032/182590764-9d0546ad-bd8b-4339-86a6-164400de92c4.png)
4. Transfromed rrggbbaa(hex) after build in production env
![14751D1B-90B3-497A-9682-801628348EAA](https://user-images.githubusercontent.com/13961032/182591524-dcd5737e-03e8-4394-ab8c-bd45a50c040b.png)
5. I just want to use rgba color instead of hex(In fact, rrggbbaa color does not working in some broswer).So Is there some way i can disable this option? THANKS

**Comment by adamwathan** [comment, marked as answer]
Hey! This is your minifier doing this (`#3d3d592e` is smaller than `rgba(61, 61, 89, 0.18)`), not Tailwind. Nothing we can do about this unfortunately, it's another tool later in your toolchain.

**Comment by Alvinnn1** [comment]
Fixed the problem when i add this option in vite.config.js. Thank u!
`build.cssTarget: 'chrome61'`
